"cover:url" fails on v2.10.2

### What happened ?

```
$ mangadex-dl "cover:https://mangadex.org/title/a6bfedbc-9846-4a97-9fcb-77a426d978e6/ubunchu" 
[ERROR] Unhandled exception, AttributeError: 'NoneType' object has no attribute 'id'
Traceback (most recent call last):
  File "./mangadex-downloader/mangadex_downloader/cli/__init__.py", line 73, in _main
    build_url(parser, args)
  File "./mangadex-downloader/mangadex_downloader/cli/url.py", line 238, in build_url
    result = command.prompt(args.input_pos)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "./mangadex-downloader/mangadex_downloader/cli/command.py", line 654, in prompt
    result = super().prompt(input_pos)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "./mangadex-downloader/mangadex_downloader/cli/command.py", line 273, in prompt
    return [answer.id]
            ^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'id'
```
I consistently get this error even with different urls.

### What did you expect to happen ?

prompt which covers to download

### OS version

Linux

### App version

mangadex-downloader v2.10.2 (https://github.com/mansuf/mangadex-downloader)
Python: 3.11.3
arch: x64
bundled executable: no

### Installation origin

git clone && python setup.py install

### Installation origin (other sources)

_No response_

### Reproducible command

`mangadex-dl "cover:https://mangadex.org/title/a6bfedbc-9846-4a97-9fcb-77a426d978e6/ubunchu"`

### Additional context

The command works when I'm checked out on older version (v2.9.0) , which was the version that introduced the "cover:" feature.

--- a/mangadex_downloader/cli/command.py
+++ b/mangadex_downloader/cli/command.py
@@ -55,6 +55,7 @@ from ..chapter import Chapter
 from ..mdlist import MangaDexList
 from ..group import Group
 from ..cover import CoverArt, cover_qualities
+from ..language import get_language
 
 def preview_chapter(chapter: Chapter):
     try:
@@ -630,9 +631,15 @@ class CoverArtCommand(MangaDexCommand):
         if quality not in cover_qualities:
             parser.error(f"{quality} is not valid quality covers")
 
+        from ..config import config
+        vcl = config.volume_cover_language
+        cover_locale = vcl if vcl else config.language
+        cover_locale = get_language(cover_locale)
+
         manga = Manga(_id=manga_id)
         iterator = CoverArtURLIterator(manga_id, quality)
         text = f"List of covers art from manga '{manga.title}' in {quality} quality"
+        text += f" ({cover_locale.name} language)"
 
         super().__init__(
             parser,
@@ -641,6 +648,8 @@ class CoverArtCommand(MangaDexCommand):
             text,
         )
 
+        self.cover_locale = cover_locale
+        self.manga = manga
         self.manga_id = manga_id
         self.quality = quality
         self.limit = 10
@@ -660,6 +669,12 @@ class CoverArtCommand(MangaDexCommand):
             return [get_cover_art_url(cover.manga_id, cover, self.quality)]
 
         return result
+
+    def on_empty_error(self):
+        self.args_parser.error(
+            f"Manga {self.manga.title!r} doesn't " \
+            f"have {self.cover_locale.name!r} covers"
+        )
 
 registered_commands = {
     "search": SearchMangaCommand,

--- a/mangadex_downloader/cover.py
+++ b/mangadex_downloader/cover.py
@@ -68,7 +68,13 @@ class CoverArt:
             pass
 
     def __str__(self) -> str:
-        return f"Cover volume {self.volume}"
+        from .config import config
+
+        msg = f"Cover volume {self.volume}"
+        if config.language == "all" or config.volume_cover_language == "all":
+            msg += f" in {self.locale.name} language"
+
+        return msg
 
     @property
     def volume(self):

--- a/mangadex_downloader/iterator.py
+++ b/mangadex_downloader/iterator.py
@@ -480,8 +480,10 @@ class CoverArtIterator(BaseIterator):
                 "manga[]": self.manga_id,
                 "offset": self.offset,
                 "limit": self.limit,
-                "locales[]": cover_locale
             }
+
+            if cover_locale != "all":
+                params.update({"locales[]": cover_locale})
 
             url = f"{base_url}/cover"
             r = Net.mangadex.get(url, params=params)

